src/app/@ansyn/menu/actions/menu.actions.ts

```diff
@@ -73,9 +93,46 @@ export class SetAutoClose implements Action {
 	}
 }
 
-export class ToggleMenuCollapse implements Action {
+export class ToggleMenuCollapse implements Action, ILogMessage {
 	type = MenuActionTypes.MENU_COLLAPSE;
 
 	constructor(public payload: boolean) {
 	}
+
+	logMessage() {
+		return `${this.payload ? '' : 'Un-'}Hiding menu`
+	}
+}
+
+export class LogOpenPermissionsSite implements Action, ILogMessage {
```

Comment:
The `@ansyn/menu` package should not know the items inside.

Reply:
@tltk90 fixed. I created a directory for credentials under menu-items